docs/source/projects/dissect.btrfs/index.rst

```diff
@@ -18,17 +18,10 @@ Installation
 
     $ pip install dissect.btrfs
 
-This project decompresses lzo compressed file systems and can use the faster, native (C-based) lzo implementation when
-installed, instead of the slower pure Python implementation provided by :doc:`/projects/dissect.util/index`. To use
-these faster implementations, install the package with the lzo extra:
-
-   .. code-block:: console
-
-    $ pip install "dissect.btrfs[lzo]"
-
-Unfortunately there is no binary ``python-lzo`` wheel for PyPy installations on Windows, so it won't be installed there.
-
-This module including the lzo extra is also automatically installed if you install the ``dissect`` package.
+This project decompresses LZO compressed file systems and can use a faster, native (Rust-based) LZO implementation,
+instead of a slower pure Python implementation. Both implementations are provided by :doc:`/projects/dissect.util/index`.
+Pre-build wheels are available for most common platforms and the native implementation will automatically be used if available.
+In the rare case that a pre-build wheel is not available, please refer to :doc:`/projects/dissect.util/index` for build instructions.
```

Comment:
Now this reads as if the whole file system is decompressed. Maybe something along the lines of:
```suggestion
This project decompresses LZO compressed file streams. For this it uses a native rust-based LZO implementation, instead of a slower pure Python implementation. Both implementations are provided by :doc:`/projects/dissect.util/index`.
Pre-build wheels are available for most common platforms and the native implementation will automatically be used if available.
In the rare case that a pre-build wheel is not available, please refer to :doc:`/projects/dissect.util/index` for build instructions.
```

Reply:
```suggestion
This project supports file systems with LZO compression. It can use either a fast Rust-based LZO implementation, or a slow pure Python implementation. Both implementations are provided by :doc:`/projects/dissect.util/index`, and the faster implementation will automatically be used if available.
Pre-build wheels are available for most common platforms. In the rare case that a pre-build wheel is not available, please refer to :doc:`/projects/dissect.util/index` for build instructions.
```